Dockerfile

```diff
@@ -22,10 +22,12 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+# Use Invenio's CentOS7 image with Python-3.6
 FROM inveniosoftware/centos7-python:3.6
 
-# Pinning to 4.12.2 because of https://github.com/xrootd/xrootd/issues/1284
+# Use XRootD 4.12.2
```

Comment:
Note: `master` keeps 4.12.2 even though `qa` had to upgrade to 4.12.6 due to respective CentOS build failure differences in base images.  

Looking forward to the day when PyPI xrootd package would be nicely installable without that `v-` bug :)